app/javascript/styles/basics.scss

```diff
@@ -61,3 +61,7 @@ button {
   align-items: center;
   justify-content: center;
 }
+
+.has-system-font {
```

Comment:
Minor nitpick, maybe something like `.system-font` would be better? `has` for me implies that we do some feature detection (e.g. `has-flex`, `has-grid`).